### PR TITLE
style(api-client): fix z index bugs, flex bugs, reduce details of mod…

### PR DESCRIPTION
**Pull request by cameronrohani**
fix 1 before:
image

fix 1 after:
image

fix 2 before:
image

fix 2 after:
image

fix 3 before:
image

fix 3: after
image

fix 4: before
image

fix 4: after
image

fix 5:
request + response body had different font sizes which I fixed

change 1 reduce details on the modal component: before
image

change 1 after:
image

change 2:  remove sidebar toggle for extra large screens 
image

change 3 reduce details of active state in card in order to align the toggle:
before:
image

after:
image

### CODE REVIEW

File: packages/api-client/src/components/ViewLayout/ViewLayoutSection.vue [hunk 1]
```diff
@@ -2,7 +2,7 @@
   <section
     class="flex h-full xl:min-w-0 xl:flex-1 flex-col xl:custom-scroll bg-b-1">
     <div
-      class="min-h-12 xl:min-h-header flex items-center border-b-1/2 px-3.5 py-1.5 md:px-4 md:py-2 xl:px-6 xl:pr-5 text-sm font-medium sticky top-0 bg-b-1 xl:rounded-none">
+      class="min-h-12 xl:min-h-header flex items-center border-b-1/2 px-3.5 py-1.5 md:px-4 md:py-2 xl:px-6 xl:pr-5 text-sm font-medium sticky top-0 bg-b-1 xl:rounded-none z-1">
```
Comment (hwkr) [inline packages/api-client/src/components/ViewLayout/ViewLayoutSection.vue:5]: z-1 ftw 🙏

File: packages/api-client/src/views/Request/RequestSection/RequestTableTooltip.vue [hunk 2]
```diff
@@ -20,7 +20,7 @@ const hasItemProperties = computed(
     class="w-full"
     :delay="0"
     side="left"
-    triggerClass="before:absolute before:content-[''] before:bg-gradient-to-r before:from-transparent before:to-b-1 before:min-h-full before:right-[23px] before:w-3 absolute h-full right-0">
+    triggerClass="before:absolute before:content-[''] before:bg-gradient-to-r before:from-transparent before:to-b-1 before:min-h-[calc(100%-4px)] before:pointer-events-none before:right-[23px] before:top-[2px] before:w-3 absolute h-full right-0">
```
Comment (hwkr) [inline packages/api-client/src/views/Request/RequestSection/RequestTableTooltip.vue:23]:
```suggestion
    triggerClass="before:absolute before:content-[''] before:bg-gradient-to-r before:from-transparent before:to-b-1 before:min-h-[calc(100%-4px)] before:pointer-events-none before:right-[23px] before:top-0.5 before:w-3 absolute h-full right-0">
```

File: packages/api-client/src/views/Request/RequestSubpageHeader.vue [hunk 3]
```diff
@@ -20,14 +21,19 @@ defineEmits<{
...
+const { layout } = useLayout()
 </script>
 <template>
   <div
     class="lg:min-h-header flex items-center w-full justify-center p-2 pt-2 lg:pt-1 lg:p-1 flex-wrap t-app__top-container border-b-1/2">
     <div
       class="flex flex-row items-center gap-1 lg:px-1 lg:mb-0 lg:mb-0 mb-2 lg:flex-1 w-6/12">
       <SidebarToggle
-        class="gitbook-hidden"
+        :class="[
```
Comment (hwkr) [inline packages/api-client/src/views/Request/RequestSubpageHeader.vue:32]: I think maybe we can avoid the importants here?

```suggestion
        class="gitbook-hidden"
        :class="[
        layout === 'modal' : 'flex' :
          modelValue ? 'xl:hidden' : 'xl:flex'
        ]"
```

**Review by hwkr — APPROVED** [review]
Looks good! A couple of little improvements / suggestions

**Comment by cameronrohani** [comment]
> Looks good! A couple of little improvements / suggestions

done and done! :)

**Review by marclave — APPROVED** [review]

File: packages/api-client/src/components/ViewLayout/ViewLayoutCollapse.vue [hunk 4]
```diff
@@ -17,11 +17,11 @@ withDefaults(
   <Disclosure
     v-slot="{ open }"
     as="div"
-    class="focus-within:bg-b-2 ui-not-open:hover:bg-b-2 focus-within:text-c-1 text-c-2 rounded request-item ui-not-open:bg-transparent ui-open:pb-1 ui-open:[&:not(:last-child)]:mb-1.5 ui-not-open:mb-0 ui-not-open:pb-0"
+    class="focus-within:bg-b-2 ui-not-open:hover:bg-b-2 focus-within:text-c-1 text-c-2 rounded request-item ui-not-open:bg-transparent ui-open:[&:not(:last-child)]:mb-2.5 ui-not-open:mb-[0.5px]"
     :defaultOpen="defaultOpen">
     <div class="flex items-center">
       <DisclosureButton
-        class="group hover:text-c-1 flex flex-1 items-center gap-2.5 overflow-hidden py-1.5 text-sm font-medium px-1.5 outline-none">
+        class="group hover:text-c-1 flex flex-1 items-center gap-2.5 overflow-hidden py-1.5 text-sm font-medium px-2.5 outline-none">
```
Comment (antlio) [inline packages/api-client/src/components/ViewLayout/ViewLayoutCollapse.vue:24]: nitpick: could use diff padding here to maintain caret alignment:

```suggestion
        class="group hover:text-c-1 flex flex-1 items-center gap-2.5 overflow-hidden py-1.5 text-sm font-medium px-1 md:px-1.5 xl:px-2 outline-none">
```

Comment (cameronrohani) [reply packages/api-client/src/components/ViewLayout/ViewLayoutCollapse.vue:24]: feel free to change it :)

**Pull request merged**